portal/portal-impl/impl/pom.xml

```diff
@@ -62,6 +62,10 @@
                 <artifactId>entitybroker-utils</artifactId>
         </dependency>
         <dependency>
+                <groupId>org.sakaiproject.profile2</groupId>
+                <artifactId>profile2-api</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
This all looks good, my only concern would be the dependency on profile2-api in the portal - is this something we want or do we want this decoupled via EB?